NOTICE

```diff
@@ -0,0 +1,130 @@
+### Open Source Software Notice ###
+
+This product from Finschia Foundation contains the open source software detailed below. Please refer to the indicated open source licenses (as are included following this notice) for the terms and conditions of their use.
+
+
+Please be informed that  Finschia Foundation   product may contain open source software listed in to tables below.
+
+Finschia/ibc-go  (MIT)
+Copyright (c) 2022 COSMOS
+
+gogo/protobuf  (BSD-3-Clause)
+Copyright 2013, The GoGo Authors
+Copyright 2010 The Go Author
+
+keplr-wallet  (Apache-2.0)
+Copyright 2020 Chainapsis, Inc
```

Comment:
Where are we using keplr-wallet in ibc-go?

Reply:
When I check it by exports, It is no need. So I remove it. Thank you for checking.